### PR TITLE
Cleanup test utils

### PR DESCRIPTION
**Pull request by seando-adsk**
Cleanup the test utils by ensuring proper separation between Maya, Ufe, Usd and general purpose.

### CODE REVIEW

**Comment by seando-adsk** [comment]
See original PR #957. As agreed with Krystian we will finish the work and then close that PR.

File: test/lib/testMayaUsdProxyAccessor.py [hunk 1]
```diff
@@ -27,7 +27,8 @@
 
 from cachingUtils import NonCachingScope, CachingScope
 from ufeUtils import createUfeSceneItem, selectUfeItems
-from mayaUtils import createProxyAndStage, createProxyFromFile, createAnimatedHierarchy
+from mayaUtils import createProxyAndStage, createProxyFromFile
+from usdUtils import createAnimatedHierarchy
```
Comment (seando-adsk) [inline test/lib/testMayaUsdProxyAccessor.py:31]: I moved the function createAnimatedHierarchy as it uses Usd.

File: test/testUtils/mayaUtils.py [hunk 2]
```diff
@@ -124,11 +121,8 @@ def getMayaSelectionList():
     else:
         return [x for x in cmds.ls(sl=True)]
 
-def getTestScene(*args):
-    return os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", "testSamples", *args)
-
```
Comment (seando-adsk) [inline test/testUtils/mayaUtils.py:129]: I moved this function into 'testUtils.py' as it doesn't need Maya.

File: test/testUtils/mayaUtils.py [hunk 3]
```diff
@@ -203,39 +197,6 @@ def createProxyFromFile(filePath):
...
-    
-    time1 = Usd.TimeCode(1.)
-    UsdGeom.XformCommonAPI(parentPrimA).SetTranslate((0,0,0),time1)
-    UsdGeom.XformCommonAPI(childPrimSphere).SetTranslate((5,0,0),time1)
-    UsdGeom.XformCommonAPI(childPrimCube).SetTranslate((0,0,5),time1)
-    
-    time2 = Usd.TimeCode(100.)
-    UsdGeom.XformCommonAPI(parentPrimA).SetTranslate((0,5,0),time2)
-    UsdGeom.XformCommonAPI(childPrimSphere).SetTranslate((-5,0,0),time2)
-    UsdGeom.XformCommonAPI(childPrimCube).SetTranslate((0,0,-5),time2)
```
Comment (seando-adsk) [inline test/testUtils/mayaUtils.py:237]: This function does not use Maya, but instead uses Usd. So I moved it.

File: test/testUtils/ufeUtils.py [hunk 4]
```diff
@@ -21,7 +21,6 @@
 """
 
 import ufe
-from maya import cmds
```
Comment (seando-adsk) [inline test/testUtils/ufeUtils.py:24]: This file does not need anything from Maya.

**Review by seando-adsk — COMMENTED** [review]
There are two versions (mayaUtils & usdUtils) of the function "createUfePathSegment". I considered moving them since they use Ufe, but they take in a Maya path and Usd path. @ppt-adsk What do you think?

**Comment by marsupial** [comment]
Working for the parent-project case in #957. I'll close out that PR.
Thanks!

**Review by ppt-adsk — APPROVED** [review]

**Pull request merged**